JP-3857 rscd code style updates (#9275)
spacetelescope/jwst: docs/jwst/rscd/description.rst

Reviewer: It looks like there are minor grammatical issues throughout the docs of this step.  I know you were not the one to create these, but it might be good to proofread the whole file.  I made suggested changes only on the lines that GitHub deems are "changed lines"

nonlinearities at the start of an integration, non-linear ramps with increasing signal,
latent images, and drifts in the slopes.

> THE MIRI readout electronics use field effect transisitors (FETs) in their opertation,

Reviewer 2: A few typos here:
```suggestion
The MIRI readout electronics use field effect transistors (FETs) in their operation,
```

Reviewer 2: @jemorrison - There are still two typos here following your update:
`THE` should be `The`
and
`opertation` should be `operation`

> The manner in which the MIRI readout electronics operate have been shown to be the source of

Reviewer: ```suggestion
The manner in which the MIRI readout electronics operate has been shown to be the source of
```

> the ramp offsets, nonlinearities at the start of the integration, and overall changes in slopes.

Reviewer: ```suggestion
the ramp offsets, the nonlinearities at the start of the integration, and the overall changes in slopes.
```

Reviewer 2: Looks like this could still be fixed too, to use "the" in front of ramp offsets, nonlinearities, and overall changes (or else remove the "the" in front of ramp offsets).

> which have been shown to be the source of the ramp offsets, nonlinearities at the start
> of an integration, and overall changes in the slopes. The FET acts as a switch to allow

Reviewer 2: ```suggestion
which have been shown to be the source of the ramp offsets, the nonlinearities at the start
of an integration, and the overall changes in the slopes. The FET acts as a switch to allow
```

> Basically the MIRI reset electronics use field effect transistors (FETs) in their operation.

Reviewer: ```suggestion
Basically, the MIRI reset electronics use field effect transistors (FETs) in their operation.
```

The FET acts as a switch to allow charge to build up and to also initialize (clear) the charge.

> However, the reset FETS do not instantaneously reset the level, instead the exponential

Reviewer: ```suggestion
However, the reset FETS do not instantaneously reset the level. Instead, the exponential
```

> adjustment of the  FET after a reset causes the initial frames in an integration to be offset

Reviewer: ```suggestion
adjustment of the FET after a reset causes the initial frames in an integration to be offset
```

> from their expected values. Between exposures the MIRI detectors are continually reset;

Reviewer: ```suggestion
from their expected values. Between exposures, the MIRI detectors are continually reset;
```

> however for a multiple integration exposure there is a single reset between integrations.

Reviewer: ```suggestion
however, for a multiple integration exposure there is a single reset between integrations.
```

> The effects of this decay are not measurable in the first integration  because a number

Reviewer: ```suggestion
The effects of this decay are not measurable in the first integration because a number
```

of resets have occurred from the last exposure and the effect has decayed away by the time

> it takes to read out the last exposure, set up the next exposure and begin exposing.

Reviewer: ```suggestion
it takes to read out the last exposure, set up the next exposure, and begin exposing.
```
